src/braket/simulator/braket_simulator.py

```diff
@@ -59,6 +62,40 @@ def run(
             representing the results of the simulation.
         """
 
+    def run_multiple(
+        self,
+        payloads: Sequence[Union[OQ3Program, AHSProgram, JaqcdProgram]],
```

Comment:
[nit] naming: `programs` or `irs` instead of `payloads`?

Reply:
Well, the fact that the input types all end with "Program", seems to support Ryan's point.